### PR TITLE
feat: Add support for new functions

### PR DESCRIPTION
**Pull request by ron-gal**
Thank you for opening a Pull Request! Before submitting your PR, there are a few things you can do to make sure it goes smoothly:
- [x] Make sure to open an issue as a [bug/issue](https://github.com/googleapis/java-bigtable/issues/new/choose) before writing your code!  That way we can discuss the change, evaluate designs, and agree on the general idea
- [x] Ensure the tests and linter pass
- [x] Code coverage does not decrease (if any source code was changed)
- [x] Appropriate docs were updated (if necessary)
- [x] Rollback plan is reviewed and LGTMed
- [x] All new data plane features have a completed end to end testing plan

Fixes #<issue_number_goes_here> ☕️

If you write sample code, please follow the [samples format](
https://github.com/GoogleCloudPlatform/java-docs-samples/blob/main/SAMPLE_FORMAT.md).

### CODE REVIEW

File: google-cloud-bigtable/src/main/java/com/google/cloud/bigtable/admin/v2/models/Type.java [hunk 1]
```diff
@@ -87,13 +92,48 @@ public static Aggregate int64Sum() {
...
+    return max(bigEndianInt64());
+  }
+
+  /** Creates an Aggregate type with a MAX aggregator and specified input type. */
+  public static <Input extends Type & MaxAggregateInput> Aggregate max(Input inputType) {
+    return Aggregate.create(inputType, Aggregate.Aggregator.Max.create());
+  }
+
+  /** Creates an Aggregate type with a HLL aggregator and Int64 input type. */
+  public static Aggregate rawBytesHll() {
```
Comment (steveniemitz) [inline google-cloud-bigtable/src/main/java/com/google/cloud/bigtable/admin/v2/models/Type.java:125]: shouldn't this be called int64Hll?

File: google-cloud-bigtable/src/main/java/com/google/cloud/bigtable/admin/v2/models/Type.java [hunk 2]
```diff
@@ -87,13 +92,48 @@ public static Aggregate int64Sum() {
...
+
+  /** Creates an Aggregate type with a HLL aggregator and specified input type. */
+  public static <Input extends Type & HllAggregateInput> Aggregate hll(Input inputType) {
+    return Aggregate.create(inputType, Aggregate.Aggregator.Hll.create());
+  }
+
   /** Represents a string of bytes with a specific encoding. */
   @AutoValue
-  public abstract static class Bytes extends Type {
+  public abstract static class Bytes extends Type implements HllAggregateInput {
```
Comment (steveniemitz) [inline google-cloud-bigtable/src/main/java/com/google/cloud/bigtable/admin/v2/models/Type.java:136]: same, we only support int64 inputs right now so we shouldn't have this here.

File: google-cloud-bigtable/src/main/java/com/google/cloud/bigtable/admin/v2/models/Type.java (hunk 1 above)
Comment (ron-gal) [reply google-cloud-bigtable/src/main/java/com/google/cloud/bigtable/admin/v2/models/Type.java]: Whoops! D= fixed!

File: google-cloud-bigtable/src/main/java/com/google/cloud/bigtable/admin/v2/models/Type.java (hunk 2 above)
Comment (ron-gal) [reply google-cloud-bigtable/src/main/java/com/google/cloud/bigtable/admin/v2/models/Type.java]: fixed

**Review by steveniemitz — APPROVED** [review]

**Pull request merged**